src/components/NameValueInput.tsx

```diff
@@ -50,7 +50,7 @@ export const NameValueInput = ({ name, disabled, limit, label, validateName, val
             `}
           >
             <Input
-              {...register(`${name}.${index}.value` as const, { required: true, validate: validateValue })}
```

Comment:
Did the validation logic move somewhere else? I don't think we want to delete it if not

Reply:
Oh, oops! Good catch! I meant to just remove the `as const` assertion. I couldn't see the need for it?

***
_Tangential rant..._

I hate the 120 character line limit we have across Grafana repos and if we used the default [80 as Prettier recommends](https://prettier.io/docs/en/options.html) I probably would have noticed everything after the assertion.

It [wasn't as chaotic](https://github.com/grafana/grafana-k6-app/pull/672) as we worried it was going to be in the k6 plugin 😅

Reply:
Yeah I remember that being annoying to me too. It's been so long I've completely forgotten. If you want to change it to 80, sally forth.